### ISSUE TITLE
req.cookies is not filled

### ISSUE THREAD

**Issue by LoganTann** [l2ig/aqua#96]
Hello,

I'm actually creating a session system for my project. I need to read and store a session id from cookie.

**Expected behavior** : if the user holds a cookie with name `session`, then a controller requesting `req.cookies.session` should hold the cookie value

**Current behavior** : the object `req.cookies` is an empty object. However, `req.headers.cookie` contains the raw data so a quickfix is easy.

**Screenshot** : 

![image](https://user-images.githubusercontent.com/28659185/177055872-c2101423-66fe-479e-9cac-5738be05db5f.png)

---

Also, when I save values in **req**.cookie, I need to use the following code in an outgoing middleware to sync with **res**.cookie

```ts
if (!res.cookies["session"]) {
    res.cookies["session"] = req.cookies["session"];
}
```

**Comment by l2ig** [comment]
Is your repo maybe public or do you have a reproducible example?

**Comment by l2ig** [comment]
But regarding the last point: Yes, setting the cookie on the request won't do anything - it's the correct behaviour.

But could maybe be made more obvious by only providing a getter - will be changed in v2.

**Comment by LoganTann** [comment]
Hello, thanks for the quick reply ! 

You can look here the code in which I added my session system and used cookies : https://github.com/LoganTann/redbeansoup-v2/commit/9b7ea5d1cd3b5cd25300c52394e5ae5a64d7cd56


If you want to run the entire repo, then use the following command line argument : `deno run --import-map import_map.json --no-check --allow-all --watch mod.ts  --start-dashboard-only`

You may need to create the file `config.ts` with the following content (no need to fill) :

```js
export default {
    // Discord Developers -> Oauth2 -> Client ID
    CLIENT_ID: "",
    // Discord Developers -> Oauth2 -> Client secret
    CLIENT_SECRET: "",
    // Discord Developers -> Bot -> reset
    BOT_TOKEN: "",
    // Right Click on your dev server icon -> Copy ID
    DEV_GUILD_ID: "",
    // Get your OpenAI token in your dashboard - https://openai.com/api/
    OPENAI_TOKEN: "",
    WHOLESOME_ENDPOINT: "",
};
```

**Comment by LoganTann** [comment]
omg you fixed it before my reply, so fast xD

thanks a lot, i'll look into that PR

**Comment by l2ig** [comment]
Thanks a lot for opening the issue! :raised_hands: Fix released with v1.3.5
